vendor/tensorflow/workspace.bzl

```diff
@@ -22,6 +22,13 @@ def plaidml_tf_workspace():
         },
     )
 
+    http_archive(
+        name = "tfhub_i3d_kinetics_400",
+        url = "https://storage.googleapis.com/tfhub-modules/deepmind/i3d-kinetics-400/1.tar.gz",
+        sha256 = "bafe29bb4528badad428207d8fe86ca2a998b5b1386b82e42175339f12ea2ff5",
+        build_file = clean_dep("//vendor/tensorflow:tfhub_i3d_kinetics_400.BUILD"),
+    )
+
```

Comment:
@dgkutnic This approach obviates the need to manage the TF hub cache; we let bazel manage it instead.